tests/examples/example8-schPinStyle-accepts-pinLabels.test.tsx

```diff
@@ -0,0 +1,22 @@
+import { test, expect } from "bun:test"
+import { getTestFixture } from "tests/fixtures/get-test-fixture"
+
+test("schPinStyle accepts pinLabels", async () => {
+  const { circuit } = getTestFixture()
+  circuit.add(
+    <board width="12mm" height="10mm">
+      <chip
+        name="U1"
+        footprint="soic8"
+        pinLabels={{ pin1: "PWR", pin8: "GND" }}
+        schPinStyle={{ PWR: { bottomMargin: 1 }, GND: { bottomMargin: 0.5 } }}
+        pcbX={0}
+        pcbY={0}
+      />
+    </board>,
+  )
+
+  circuit.render()
+
+  await expect(circuit).toMatchSchematicSnapshot(import.meta.path)
```

Comment:
Can you add a pinlabels array test here as well? Generally you should try to test every code path, i suspect there may be a bug with pin label arrays based on your implementation

Reply:
sure thing : )

Reply:
For your test, try to test (in a single test) variations on the usage, eg the pin label could be the first entry)

i also suspect you may not handle the case where schPinStyle contains a pin label that doesnt exist, but this can be a future issue.

based on your code, i think there would be a bug if schPinStyle has a substring in the key, eg if it had a key “GND” but the pinlabels array listed “GND1” , “GND2”

Reply:
Gotcha will update for the same